pkg/deploy/api/v1beta1/types.go

```diff
@@ -55,6 +55,8 @@ type DeploymentStrategy struct {
 	CustomParams *CustomDeploymentStrategyParams `json:"customParams,omitempty"`
 	// RecreateParams are the input to the Recreate deployment strategy.
 	RecreateParams *RecreateDeploymentStrategyParams `json:"recreateParams,omitempty"`
+	// Compute resource requirements to execute the deployment
+	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
See long argument that was settled here: 

https://github.com/openshift/origin/pull/1821/files#r28747341

Result: be consistent with upstream.